proxy/http/HttpConfig.cc

```diff
@@ -1224,12 +1224,16 @@ HttpConfig::reconfigure()
 
   params->oride.connect_attempts_max_retries             = m_master.oride.connect_attempts_max_retries;
   params->oride.connect_attempts_max_retries_dead_server = m_master.oride.connect_attempts_max_retries_dead_server;
-  params->oride.connect_attempts_rr_retries              = m_master.oride.connect_attempts_rr_retries;
-  params->oride.connect_attempts_timeout                 = m_master.oride.connect_attempts_timeout;
-  params->oride.post_connect_attempts_timeout            = m_master.oride.post_connect_attempts_timeout;
-  params->oride.parent_connect_attempts                  = m_master.oride.parent_connect_attempts;
-  params->per_parent_connect_attempts                    = m_master.per_parent_connect_attempts;
-  params->parent_connect_timeout                         = m_master.parent_connect_timeout;
+  if (m_master.oride.connect_attempts_rr_retries >= params->oride.connect_attempts_max_retries) {
+    Warning("connect_attempts_rr_retries is greater than connect_attempts_max_retries, this means requests will never redispatch "
```

Comment:
I think the specific destinction is quite important, its possible that you'd want to *only* dispatch (which we can do with these 2 config options as they stand). Sadly these terms aren't terribly familiar with people in ATS because, until recently, it has been broken ;)